src/system/Wizard/WizardStep.tsx

```diff
@@ -53,7 +73,7 @@ const WizardStep = React.forwardRef(
 		const statusIconColor = `wizard.step.icon.${ status }`;
 		const headingColor = `wizard.step.heading.${ status }`;
 
-		const StatusIcon = complete ? BsFillCheckCircleFill : BsCircleFill;
+		const StatusIcon: IconType = complete ? BsFillCheckCircleFill : BsCircleFill;
```

Comment:
I am a little stumped by this one, need to think about it.

Reply:
removed, I was confused too, but I think IntelliJ was playing tricks on my. Running `tsc` didn't bring up errors. I removed it in https://github.com/Automattic/vip-design-system/pull/251/commits/37f0c76fb59257386d87703b9b60844601214f80

Reply:
@chriszarate it's not a TS problem, but more of the 'no-unsafe-assignment' eslint rule (which was failing even with or without the type)
Adding back the IconType type doesn't really help, as for some reasons it's seeing an "any" assignment.

To fix this I've moved the ternary switch to the components part and it now passes the linting, but I'm sure will find this situation again, so if you have a better/different approach I'd love to hear it.

fix: https://github.com/Automattic/vip-design-system/pull/251/commits/f4277fa8349a8bd14fd4117b14ad3663b42ff07b

Reply:
Nice, we were wrestling with the same issue in `Notice` and didn't think to just sidestep assignment. Thanks!